addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -6,6 +6,9 @@
         <t t-set="navbar__back_url" t-value="'/event/%s/track' % (slug(event))"/>
         <t t-set="navbar__back_title">Back to All Talks</t>
         <t t-set="navbar__back_text">All Talks</t>
+        <!-- Set aside block only if there are filtered tracks -->
+        <t t-set="has_ongoing_or_upcoming" t-value="any(track.is_track_live or track.track_start_remaining for track in tracks_other)"/>
```

Comment:
Question: not sure if it was on purpose but is it normal that the live track are display last in the side track menu on a track page ?
![image](https://github.com/user-attachments/assets/dab7e6b1-473a-43f6-a98d-4bffcc64ba25)

Reply:
Well not sure what happens, seems resolve on my end as well. You can ignore it then :shrug: